JSDemos/Demos/TreeList/UsingFilterRow/description.md

```diff
@@ -1 +1,30 @@
-This demo shows how you can filter data using the filter row. Its cells allow filtering by&nbsp;individual columns. Select or&nbsp;type a&nbsp;value into a&nbsp;cell of&nbsp;the filter row. If&nbsp;this value is&nbsp;found in&nbsp;any of&nbsp;the rows, the TreeList displays the full hierarchy down to&nbsp;them, but not further. In&nbsp;addition, you can choose the current filter operation from the drop-down menu that opens when you hover over the magnifying glass icon.
\ No newline at end of file
+The filter row allows users to enter a query for each column and filter rows by that value. To display the filter row, set the **filterRow**.[visible](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/filterRow/#visible) property to **true**. If you need to deactivate this feature for a certain column, set its [allowFiltering](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowFiltering) property to **false**.
+
+Depending on a column's [dataType](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#dataType), its filter row cell represents different editors:
+
+<table class="dx-table">
+    <tr>
+        <th>dataType</th>
+        <th>Editor</th>
+    </tr>
+    <tr>
+        <td>*"string"*, *"number"*, *"object"*</td>
+        <td>text box</td>
+    </tr>
+    <tr>
+        <td>*"boolean"*</td>
+        <td>drop-down list</td>
+    </tr> 
+    <tr>
+        <td>*"date"*</td>
+        <td>date picker</td>
+    </tr> 
+    <tr>
+        <td>*"datetime"*</td>
+        <td>date and time picker</td>
+    </tr> 
+</table>
+
+Users can click a magnifying glass icon in the filter row cell to select a different filter type. These filters and their initial settings are also defined by the column's data type. If you need to restrict the available filters, specify them in the **columns[]**.[filterOperations](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#filterOperations) array. You can also use the [selectedFilterOperation](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#selectedFilterOperation) property to specify a custom initial filter operation.
```

Comment:
I restructured this paragraph to connect it with the previous one. Also, I replaced "filter" with "filter operation" because the term "filter" (aka "filter expression") includes the data field being filtered, the filter operation, and the value to filter by.

```suggestion
Each data type supports its own set of filter operations. Users can click a magnifying glass icon in each filter row cell to specify the [selectedFilterOperation](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#selectedFilterOperation). If you need to restrict the available operations, specify the **columns[]**.[filterOperations](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#filterOperations) array.
```